### DISCUSSION TITLE
Rendering *.ipynb with terminal only (no quarto settings inside notebook)

### DISCUSSION THREAD

**Discussion by stefansimik** [quarto-dev/quarto-cli#4078] [Q&A] [answered]
Hi,
I would like to ask, if it is possible to render any *.ipynb file with quarto and specify the setting in command line?

For example - what I need to achieve:
1. Lets have any standard *.ipynb, that does not have the initial section with quarto settings
2. Let's use the terminal only, to export that *.ipynb file with nicely setup Table of content
3. In general, I am interested how to pass all the Quarto options via terminal only.

I tried to Google, checked docs, but could not find such information.
Thank you in advance

Best
Stefan

**Comment by cscheid** [comment, marked as answer]
For a single option:

```
quarto render file.ipynb -M echo:false
```

For multiple options, use a file:

```
quarto render file.ipynb --metadata-file your-options.yml
```

**Comment by stefansimik** [comment]
Thank you very much, excellent solution.
